wallet/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     }
     implementation 'com.github.tony19:logback-android-core:1.1.1-6'
     implementation 'com.google.code.findbugs:jsr305:3.0.2'
+    implementation 'org.greenrobot:eventbus:3.1.1'
```

Comment:
what does eventbus do?

Reply:
It is used to send progress event to activities
http://greenrobot.org/eventbus/documentation/configuration/sticky-events/